dart/lib/src/environment/_io_environment_variables.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:io';
+
+import 'environment_variables.dart';
+import 'keys.dart';
+
+final EnvironmentVariables envs = IoEnvironmentVariables();
+
+/// In addition to dart defines this io implementation can read from the
+/// environment variables.
+class IoEnvironmentVariables extends EnvironmentVariables {
+  @override
+  String? get environment {
+    if (const bool.hasEnvironment(sentryEnvironment)) {
+      return const String.fromEnvironment(sentryEnvironment);
+    }
```

Comment:
not sure if it makes sense or if it makes even more confusing, but we could simply do like:

```
final web = WebEnvironmentVariables()

String? get environment {
  return web.environment ?? Platform.environment[sentryEnvironment];
}
```

Reply:
Good point